docs/catalogue/cat_network-data-manager.md

```diff
@@ -47,123 +44,98 @@ section [Upload metadata](cat_network-data-manager.md#upload-metadata) to the Da
+and 'Collection events' are filled out through the same route, by accessing the corresponding tables. Make sure to 
+choose the right <b>model</b> id under resource when defining your 'Subcohorts' and 'Collection events', e.g. LongITools
+should select LongITools_CDM.
 
-| *Column name* | *Description* | *Remarks* |
-| --- | --- | --- |
-| resource \* | Name of the network's model | e.g LifeCycle_CDM, LongITools_CDM or ATHLETE_CDM |
-| version \* | Version of the target data dictionary | e.g. 1.1 |
 
-<sup>Table 1. Description of the columns that can be filled out for TargetDataDictionaries. * = mandatory</sup>
+### Define the common data model
 
-#### *TargetTables* sheet
+#### *Datasets* sheet
 
-The cohort tables are defined in the *SourceTables* sheet. Columns with an asterisk (\*) after their name are mandatory.
+The network's datasets are defined in the *Datasets* sheet. Columns with an asterisk (\*) after their name are mandatory.
 
 | *Column name* | *Description* | *Remarks* |
 | --- | --- | --- |
-| dataDictionary.resource \* | Name of the model. | e.g LifeCycle_CDM, LongITools_CDM or ATHLETE_CDM |
-| dataDictionary.version \* | Target data dictionary version this table is part of. | e.g. 1.0.1 |
-| name \* | Unique table name | |
-| label | Table label | |
-| description | Table description | |
+| resource \* | Id of the <b>model</b>. | e.g LifeCycle_CDM, LongITools_CDM or ATHLETE_CDM |
+| name \* | Unique dataset name | |
+| label | Dataset label | |
+| description | Dataset description | |
 
-<sup>Table 2. Description of the columns that can be filled out for TargetTables. * = mandatory</sup>
+<sup>Table 1. Description of the columns that can be filled out for Datasets. * = mandatory</sup>
 
-#### *TargetVariables* sheet
+#### *Variables* sheet
 
-The cohort variables are defined in the *TargetVariables* sheet.
+The network's variables are defined in the *Variables* sheet.
 
 | *Column name* | *Description* | *Remarks* |
 | --- | --- | --- |
-| dataDictionary.resource \* | Name of the model that contains this variable | e.g LifeCycle_CDM, LongITools_CDM or ATHLETE_CDM |
-| dataDictionary.version \* | Target data dictionary version this variable is part of | e.g. 1.0.1 |
-| table \* | Table that contains the variable | Tables must be predefined in the TargetTables sheet |
-| name \* | Variable name, unique within a table | |
+| resource \* | Id of the <b>model</b> that contains this variable | e.g LifeCycle_CDM, LongITools_CDM or ATHLETE_CDM |
+| dataset \* | Dataset that contains the variable | Datasets must be predefined in the _Datasets_ sheet |
+| name \* | Variable name, unique within a dataset | |
 | label | Human readable variable label | |
 | format | The data type of the variable | Find list to choose from in CatalogueOntologies |
 | unit<sup>1</sup> | Unit in case of a continuous or integer format | Find list to choose from in CatalogueOntologies |
```

Comment:
missing columns: references, permitted value, order, repeats, notes,

Reply:
repeats are not to be filled out here but are rendered automatically in the catalogue. the other columns are not explained, bt also not necessary to fill out. May explain them later on.